src/channel.ts

```diff
@@ -1433,6 +1433,12 @@ export class Channel<
     );
 
     this.listeners[key].push(callback);
+
+    return {
+      unsubscribe: () => {
+        this.listeners[key] = this.listeners[key].filter((el) => el !== callback);
+      },
```

Comment:
should we call `this.off` or put the logger message to stay consistent?

Reply:
Let's go with logger message. We have `unsubscribe` on `client.on`, so lets be consistent in naming!!